### PR TITLE
Troy-Adding Minimap in hubworld

### PR DESCRIPTION
**Pull request by TroyDexter**

### CODE REVIEW

**Review by markfilliph — APPROVED** [review]
approved

**Review by ValeriaRincon — APPROVED** [review]
Looks great

**Review by Steven-David215 — APPROVED** [review]
lgtm

**Review by Seanlegroulx — APPROVED** [review]
all good

File: Assets/Scripts/MainGame/MinimapBaygate.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using System.Collections;
```
Comment (DoopweeSLC) [inline Assets/Scripts/MainGame/MinimapBaygate.cs:1]: these location minimap scripts all have the exact same code just call a different location from the enum. I'd consider making a single class like minimapLocator, and then just have a single public variable exposed for the editor to set the area to go show.

**Pull request merged**